doc/ref/peer.rst

```diff
@@ -97,6 +98,40 @@ to the manage and jobs runner functions.
         - jobs.*
 
 .. note::
+
+    Functions are matched using regular expressions.
+
+.. _peer-wheel:
+
+Peer Wheel Communication
+=========================
+
+.. versionadded: Fluorine
+
+Configuration to allow minions to execute wheel functions from the master is
+done via the ``peer_wheel`` option on the master. The ``peer_wheel``
+configuration follows the same logic as the ``peer`` option. The only
+difference is that access is granted to wheel modules.
+
+To open up access to all minions to all wheel:
```

Comment:
I still think there should be a more defined security note about the implications of this setup.

Reply:
Yes, very much so! We need to be clear that this allows the minion to alter the master configuration etc.

Reply:
done